catboost/libs/metrics/ut/fairloss_ut.cpp

```diff
@@ -0,0 +1,53 @@
+#include <library/unittest/registar.h>
+#include <catboost/libs/metrics/metric.h>
+#include <catboost/libs/metrics/metric_holder.h>
+
+Y_UNIT_TEST_SUITE(FairLossMetricTest) {
+        Y_UNIT_TEST(FairLossTest) {
+            {
+                TVector <TVector<double>> approx{{1.0, 2, 3}};
```

Comment:
unfortunately these unit tests need to be rewritten, you should create metric evaluator through `CreateMetricsFromDescription` factory function; seed `*DCG` metric tests for example (`catboost/libs/metrics/ut/dcg_ut.cpp`)